From beda2941e8bb6c90f53ba7a791b83db245a69fd6 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Wed, 7 Oct 2015 14:02:11 -0500
Subject: [PATCH 106/131] arch/arm64: Axxia Device Tree Update for XLF

As there are more than 16 cores, more memory is needed for
the redistributor registers.

Signed-off-by: John Jacques <john.jacques@intel.com>
[Original patch is from linux-yocto-4.1 axxia/base branch.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm64/boot/dts/intel/axc67xx.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/intel/axc67xx.dtsi b/arch/arm64/boot/dts/intel/axc67xx.dtsi
index 81341c0..6ddff94 100644
--- a/arch/arm64/boot/dts/intel/axc67xx.dtsi
+++ b/arch/arm64/boot/dts/intel/axc67xx.dtsi
@@ -68,7 +68,7 @@
 		#address-cells = <0>;
 		interrupt-controller;
 		reg = <0x80 0x10000000 0x10000>,
-		      <0x80 0x10400000 0x200000>;
+		      <0x80 0x10400000 0x400000>;
 		interrupts = <GIC_PPI 9 IRQ_TYPE_LEVEL_HIGH>;
 	};
 
-- 
2.8.1

